docs/source/guides/user/chapters/introduction.rst

```diff
@@ -527,5 +527,4 @@ False by default.
 
 .. warning:: If you are using Avocado from sources, you need to manually place
    the ``commands``/``fail_commands``/``fail_files``/``files``/``profilers``
-   into the ``/etc/avocado/sysinfo`` directories or adjust the paths in
-   ``$AVOCADO_SRC/etc/avocado/avocado.conf``.
```

Comment:
I am not sure if I understand why this is wrong. Because adjusting the paths in `avocado.conf` file is still a valid solution for changing the sysinfo configuration.

Reply:
I removed it because the configuration file at the above location ("$AVOCADO_SRC/...") does not exist anymore in the tree. We could keep this suggestion if I rephrase it, e.g. "adjust the paths in your ``avocado.conf``". Shall I update like this?

Reply:
Did this now and removed a superfluous "the". Branch is updated. Please pull.